sources/packages/backend/libs/sims-db/src/entities/sfas-application.model.ts

```diff
@@ -367,4 +379,48 @@ export class SFASApplication extends BaseSFASApplicationModel {
     nullable: true,
   })
   withdrawalActiveFlag?: string;
+
+  /**
+   * Student application dependants.
+   */
+  @OneToMany(
+    () => SFASApplicationDependant,
+    (dependant) => dependant.application,
+    {
+      eager: false,
+      cascade: false,
```

Comment:
`cascade: false` is the default. We have been bouncing between having and not having defaults explicitly set. I would recommend removing them.

Reply:
`eager` can also be omitted.